doc/source/ray-air/examples/convert_existing_pytorch_code_to_ray_air.ipynb

```diff
@@ -1316,7 +1315,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.8.13"
```

Comment:
doesn't look like theres actually any code changes to this file. can we revert these changes if not?